docs/tests/typo3.bats

```diff
@@ -15,7 +15,7 @@ teardown() {
   PROJNAME=my-typo3-site
   run mkdir -p ${PROJNAME} && cd ${PROJNAME}
   assert_success
-  run ddev config --project-type=typo3 --docroot=public --php-version=8.3
+  run ddev config --project-type=typo3 --docroot=public --php-version=8.4
```

Comment:
This should match the choice made in quickstart.md (keep 8.4 if 8.4 is supported)